entrypoint.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+set -e
+
+: ${DATABASE_HOST:='127.0.0.1'}
```

Comment:
Another PR added a `DATABASE_PORT` env variable. Could you please add that here?